src/xdocs/beginning_development.xml

```diff
@@ -51,7 +51,7 @@
 
       <p>
         Follow these instructions of Git usage and creating a Pull Request:<br/>
-           1) Configure remotes:
+           1) Configure remotes by setting up a connection to the official checkstyle repository, naming it "upstream":
```

Comment:
```suggestion
           1) Configure remotes by pointing to the official checkstyle repository, naming it "upstream":
```

I agree it is by convention, that we use "upstream" for remote and "origin" for fork. But I don't think we should force this in our docs. @romani , please reconsider this.

IMO, we can change : 
```suggestion
           1) Configure remotes by pointing to the official checkstyle repository:
```
to add explaination but that's it. Most VCS users use "upstream" by default (here so no reason to mention) and 
few don't (here mentioning would be wrong). In either case `, naming it "upstream"` part seems reduntant.

Reply:
>But I don't think we should force this in our docs.

We should. There are two types of contributors: who desperately need this page; who are smart enough and do not need this page or can glance at it and do it in its own way.

This page is for first type of contributors. The less option to choose the better.